Gemfile

```diff
@@ -3,3 +3,6 @@ source 'https://rubygems.org'
 gem 'rake'
 gem 'rspec'
 
+group :test, :development do
```

Comment:
We could include the other two as well! Since all dependencies are just for development/testing

Reply:
will do